src/inet/IPEndPointBasis.h

```diff
@@ -161,12 +161,12 @@ class DLL_EXPORT IPEndPointBasis : public EndPointBasis
     dispatch_queue_t mDispatchQueue;
     dispatch_semaphore_t mSendSemaphore;
 
-    INET_ERROR Bind(IPAddressType aAddressType, IPAddress aAddress, uint16_t aPort, nw_parameters_t aParameters);
+    INET_ERROR Bind(IPAddressType aAddressType, IPAddress aAddress, uint16_t aPort, nw_parameters_t & aParameters);
```

Comment:
See prior comments about passing the address by constant reference.

Reply:
Done.